Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/OptionDetails.vue

```diff
@@ -9,6 +9,10 @@ export default class OptionDetails extends Vue {
     @Prop({ required: false, default: false }) hasNotSureOption!: boolean;
     @Prop({ required: false, default: false })
     showMessageWhenNoIsSelected!: boolean;
+    @Prop({ required: false, default: false })
+    showMessageWhenYesIsSelected!: boolean;
+    @Prop({ required: false, default: false })
+    hasSelectedYesWithNoBenifit!: boolean;
```

Comment:
I think we should rename these props now that the criteria is more complicated.

```
responseOfNoIndicatesNoBenefit
responseOfYesIndicatesBenefit
responseOfYesIndicatesNoBenefit
```

I think we only use the first 2 and last one.  But it would make it easier to understand which questions had which messages.

Reply:
@MikeLyttle  I will rename the props but need to do round of test again.

Reply:
@MikeLyttle done changes. If we can avoid changing again the props name, the props name are very self-explanatory. I am testing application with different scenarios.